### ISSUE TITLE
test truncation

### ISSUE THREAD

**Issue by silvioOlivastri** [facebookresearch/fastText#361]
Hi,

I try to optimize my validation set scores with FastText supervised. I use `hyperopt` (python) to do this, so I get the output of FastText from c++ version using 

`$ ./fasttext test my_model.bin my_validation.txt`

My ""problem"" is that you truncate the output on the third digit [here](https://github.com/facebookresearch/fastText/blob/f10ec1faea1605d40fdb79fe472cc2204f3d584c/src/fasttext.cc#L375)

Is it possible get more digits?

**Comment by cpuhrsch** [comment]
Hello @silvioOlivastri,

If you are open to using our Python bindings you could train within Python directly. There you can use the fastText.util.test method that will return to you these numbers in arbitrary precision. The other advantage is also that you're not forced to save the model. This should help you speed up your hyper-parameter exploration as well. Please let me know if this is useful.

Thanks,
Christian
